main/main.cpp

```diff
@@ -3441,6 +3461,25 @@ void XtoSYS(const char* topicOri, JsonObject& SYSdata) { // json object decoding
       mqttSetupPending = true; // trigger reconnect in loop using the new topic/name
     }
 
+#ifdef BLEDecryptor
+    if (SYSdata.containsKey("ble_aes") || SYSdata.containsKey("ble_aes_keys")) {
+      if (SYSdata.containsKey("ble_aes")) {
+        strncpy(ble_aes, SYSdata["ble_aes"], parameters_size);
+      }
+      if (SYSdata.containsKey("ble_aes_keys")) {
+        Log.warning(F("Contains ble_aes_keys" CR));
+        ble_aes_keys = SYSdata["ble_aes_keys"];
+      } else {
+        // If no keys are passed clear the object.
+        StaticJsonDocument<JSON_BLE_AES_CUSTOM_KEYS> jsonBLEBuffer;
+        ble_aes_keys = jsonBLEBuffer.to<JsonObject>();
+      }
+#  ifndef ESPWifiManualSetup
+      saveConfig();
```

Comment:
This is where the `saveConfig()` function is called @DigiH so as long as you don't have `ESPWifiManualSetup` then it should work fine.

Reply:
@plambrechtsen  I overlooked that comment before, but that is what Iand many other users  have, plus me also the parent MQTT_HTTPS_FW_UPDATE undefined ;)